Jenkinsfile

```diff
@@ -95,9 +95,9 @@ pipeline {
               pythonTasksGen = new PythonParallelTaskGenerator(
                 xKey: 'PYTHON_VERSION',
                 yKey: 'FRAMEWORK',
-                xFile: "./tests/.jenkins_python.yml",
```

Comment:
all the YAML Jenkins related files are in the .ci folder, so we have to move these paths to point to the moved files